### DISCUSSION TITLE
InternetTest v7.7.1.2308

### DISCUSSION THREAD

**Discussion by lpeyr** [Leo-Corporation/InternetTest#472] [Releases]
# What's new?
## Changelog
_This update is a hotfix doesn't bring new features._

### Updated
- d68313a7964298ffa13cfab081a445b0ebc8a7d8 Updated Simplified Chinese translation (#470 by @wcxu21)

## Files
You will find these files with this release:
- InternetTestProSetup.exe

<hr /><em>This discussion was created from the release <a href='https://github.com/Leo-Corporation/InternetTest/releases/tag/7.7.1.2308'>InternetTest v7.7.1.2308</a>.</em>

**Comment by Tb3986** [comment]
Thanks for your software but it doesn't works at all. Automatically will close instantly when i run it. I tried different time even on two virtual machines clean and yes, i installed .NET 6.0 SDK.
Any tips? Same issue with previously build.
Thanks

**Reply by lpeyr** [reply]
Hi,
If this issue is happening while launching the app for the first time, then don't worry, there is indeed a crash issue. I've identified it and will be fixed in 7.7.2.2308.
If this is still not fixed, you may want to erase the app storage folder located in `"%APPDATA%\Léo Corporation\InternetTest Pro"`
Let me know if you are still encountering this problem.
Thank you for reporting this issue!

**Reply by lpeyr** [reply]
Update: It seems that another issue is causing the app to crash related to the new WiFi Networks feature. I'm investigating it.

**Reply by lpeyr** [reply]
The issue you encountered was caused by the WiFi page scanning networks: I didn't implement the case where there is no interface available or no possibility to scan nearby networks, thus causing a crash issue. Now with v7.7.3.2308, this should hopefully be fixed and you should encounter no issue (I also tested it on VMs, and now it works). #477

**Reply by Tb3986** [reply]
Great, thank you!
